src/main/java/bio/terra/service/filedata/flight/ingest/IngestDriverStep.java

```diff
@@ -143,11 +143,13 @@ public StepResult undoStep(FlightContext context) {
     private void waitForAny(FlightContext context, UUID loadId, int concurrentLoads, int originallyRunning)
         throws DatabaseOperationException, InterruptedException {
         while (true) {
-            waiting();
+            // Is evaluating the  state of the load candidates is not instant,
+            // we recheck right away before waiting only wait if we did not find any new completions3
```

Comment:
I'm having a hard time understanding this comment

Reply:
Grumble! This voice dictation is a real pita